### ISSUE TITLE
rename `walltime` to `duration`?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [uzh/gc3pie#310]
```
1) What part of the model would need changes:

This is basically just a rename, so it does not impact the model --
just the variables names and the user-level messages.

It will introduce a breaking change in the on-disk format though.


2) What is the reason why the changes is proposed:

"walltime" is HPC jargon and makes little sense otherwise:

- it *literally* makes no sense, because it's just an abbreviation for 
"wall-clock time";
- the expression "wall-clock time" itself is only defined in opposition to "CPU 
time", which is a cumulative measure of the total time spent by all processors 
in a parallel job.

Basically, the "walltime" is just measuring the time lapse between the
start of a job and its end. In other words, its duration.


3) What is the proposal:

Let's use the word "duration" throughout, whose definition can be
looked up in any dictionary.


```

Original issue reported on code.google.com by `riccardo.murri@gmail.com` on 5 Sep 2012 at 10:00
